File: social-service/src/main/java/com/eirsteir/coffeewithme/social/web/request/FriendRequest.java
package com.eirsteir.coffeewithme.social.web.request;

import javax.validation.constraints.NotNull;
import lombok.*;
import lombok.experimental.Accessors;

@Getter
@Setter
@ToString
@Builder
@Accessors(chain = true)
@NoArgsConstructor
@AllArgsConstructor
public class FriendRequest {

  @NotNull(message = "Must be a valid id")
  private Long requesterId;

  @NotNull(message = "Must be a valid id")
  private Long addresseeId;
}

File: social-service/src/main/kotlin/com/eirsteir/coffeewithme/social/web/request/FriendRequest.kt
package com.eirsteir.coffeewithme.social.web.request

import javax.validation.constraints.NotNull

data class FriendRequest(
    @get:NotNull(message = "Must be a valid id")
    val requesterId:  Long,
    @get:NotNull(message = "Must be a valid id")
    val addresseeId:  Long
)